concordium-sdk/src/main/java/com/concordium/sdk/responses/accountinfo/Baker.java

```diff
@@ -8,21 +10,40 @@
 @Getter
 @ToString
 public final class Baker {
-    private final int bakerId;
-    private final String stakedAmount;
+    /**
+     * The baker id.
+     * Note. The baker id is non-negative.
+     */
+    private final UInt64 bakerId;
+    /**
+     * The staked amount.
+     */
+    private final CCDAmount stakedAmount;
+    /**
+     * Whether earnings should be restaked automatically or not.
+     */
     private final boolean restakeEarnings;
+    /**
+     * The baker's public VRF key used to verify that the baker has won the lottery.
+     */
     private final String bakerElectionVerifyKey;
+    /**
+     * The baker's public key, used to verify baker's signatures on the blocks and finalization messages.
+     */
     private final String bakerSignatureVerifyKey;
+    /**
+     * The baker's public key used to verify the baker's signature on finalization records in case the baker is a finalizer.
+     */
     private final String bakerAggregationVerifyKey;
```

Comment:
Same as in many other places, it would be good if this was a byte array instead of String. String is just a transfer format.